src/tiles/grid/icons/account.ts

```diff
@@ -0,0 +1,11 @@
+import { html } from 'lit';
+
+export default html`
+  <svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could you add a [`<title>`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) tag inside the svg files? It's how you add screen-reader accessible info to an svg image. There's also the [`<desc>`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/desc) but that's mainly for more complex images.